### DISCUSSION TITLE
Physics3D disable an rigidbody temporary

### DISCUSSION THREAD

**Discussion by cchuicchui** [axmolengine/axmol#871] [Q&A] [answered]
In Physics3D is it possible for me to disable a rigidbody/object temporary such that raycast will not choose that object?

**Comment by aismann** [comment]
setEnabled(false) ?
Maybe reading the API function is helping mostly without any Q&A like this.

**Comment by cchuicchui** [comment, marked as answer]
I did search document and tried setActive() but it does not work.

component->setEnable(false) works well. but u cannot call setEnable at the time u created the component otherwise null pointer exception will be thrown.

Thanks.
